modules/rosapps/applications/cmdutils/vfdcmd/vfdmsg.mc

```diff
@@ -11,6 +11,11 @@
 ;#ifndef _VFDMSG_H_
 ;#define _VFDMSG_H_
 ;
+;/*
+; __REACTOS__:
+; - Removed Japanese language.
+; + Added a second blank line between some entries.
```

Comment:
Why don't we simply strip the added blank lines again, instead of documenting that (useless?) diff?

Reply:
I'm fine with keeping that "formatting" diff. Feel free to follow-up if you can't stand it.